CHANGELOG.md

```diff
@@ -47,6 +47,9 @@ a release.
 - In order to use a custom cache for storing configuration of an extension, the user has to call `setCacheItemPool()`
   on the extension listener passing an instance of `Psr\Cache\CacheItemPoolInterface`.
 
+### Deprecated
+- Using YAML mapping is deprecated, you SHOULD migrate to attributes or annotations.
```

Comment:
IMO, as we aren't deprecating the XML mapping yet, we should also add this option to the alternatives list.